### DISCUSSION TITLE
How to control the LOAD_STATE by MQTT

### DISCUSSION THREAD

**Discussion by HauWii** [softwarecrash/EPEver2MQTT#41] [Q&A] [answered]
Hello,

in my config the EPEver2MQTT has MQTT Topic "pv/epever/tracer2206an/1".
With MQTT Explorer i tried to publish "pv/epever/tracer2206an/1/EP_1/DeviceControl/LOAD_STATE" with value "0" but it don't reset (or set with "1") the LOAD.

What could be my problem why this is not working?

Thanks a lot...

**Comment by softwarecrash** [comment, marked as answer]
na wir können auch bei deutsch bleiben.
auch hier hilft das wiki evt weiter
https://github.com/softwarecrash/EPEver2MQTT/wiki/Output-Control
im mqtt nimmst du statt 0 oder 1 true oder false

**Reply by HauWii** [reply]
Manchmal ist die Lösung so nah und doch so fern...Danke vielmals !!!
